doc/dev/main.xml

```diff
@@ -9,7 +9,7 @@
 [<!ENTITY testing "<K>testing</K>">]
 >
 
-<Book Name="Development">
+<Book Name="dev">
```

Comment:
Makes sense - e.g. we use `<Book Name="tut">` in Tutorial etc. Good when book name includes in cross-references.